src/service_args.py

```diff
@@ -8,8 +8,9 @@
 
 class ServiceArgs():
 
-    def __init__(self, service_args: str):
+    def __init__(self, service_args: str, relay_rpc_urls: dict):
```

Comment:
I'm not very happy with this solution. But could not come up with something more beautiful but still easy. We might probably need to do some insurgency to this class anyway when we bring in snap support.